### PR TITLE
[RETURN-8923] Adding PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by pabloadell**

### CODE REVIEW

**Review by bertocq — COMMENTED** [review]
Same suggestion as for https://github.com/returnly/rubocop-returnly/pull/8/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R1-R16

This is a gem, we should enforce the creation of a PR on every single app that makes use of this one (returns-center and dashboard)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Ticket 🎫
+
+:link: [Jira ticket](https://returnly.atlassian.net/browse/RETURN-000)
+
+<!-- Provide a brief description of the change, its impact and the settings needed for it to work if any -->
+# Context 📓
+
+<!-- If the code cannot be tested, provide a brief description of the reasons -->
+# Testing 🧪 
+
```
Comment (pabloadell) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
# Client applications PR's 📎
- :link: [Returns center](https://github.com/returnly/returns-center/pull/XXXX)
- :link: [Dashboard](https://github.com/returnly/dashboard/pull/XXXX)
```

Comment (pabloadell) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
# Client applications PR's 📎
- :link: [Returns center](https://github.com/returnly/returns-center/pull/XXXX)
- :link: [Dashboard](https://github.com/returnly/dashboard/pull/XXXX)

```

**Review by bertocq — APPROVED** [review]
Thank you @pabloadell for pushing this! 👏

**Pull request merged**